### ISSUE TITLE
Option -f with 2 factor authentication ->asks for password but not for verification code

### ISSUE THREAD

**Issue by baalkor** [lavv17/lftp#744]
Hello,

We've faced recently an issue with lftp. We have people connecting to a SFTP server using password and verification code (pam.d/sshd => google auth).
Interactively, when we open a session it works fine  : 

$> lftp sftp://username@host
Password:
Verification code

We need to automate the transfert and be able to create a script file containing some download command.
For an unkown reason 

$> cat test-cmds.txt
ls
pwd

$> lftp sftp://username@host -f test-cmds.txt
Password:

It doesn't ask for the verification code at all and then no connection is done.

We've tested unsucessfully : 
$> lftp -u username, pass sftp://hosts
$> lftp sftp://username@host -e 'set sftp:connect-program "ssh -a -x"
$> lftp sftp://username@host -e 'set sftp:connect-program "sftp"

TLDR;  Why password and verification codes are both requested in interactive mode while only password is requested in scripting mode (-f). Is there a way we can trigger this verification code request manuall ?

**Comment by lavv17** [comment]
AFAIK lftp doesn't ask for verification code at all. It must be a feature
of ssh.

   Alexander.
